overlord/overlord.go

```diff
@@ -142,6 +142,7 @@ func New() (*Overlord, error) {
 	// setting up the store
 	authContext := auth.NewAuthContext(s, o.deviceMgr)
 	sto := storeNew(nil, authContext)
+	sto.SetCacheDownloads(5)
```

Comment:
If I understand correctly, `5` is a default value. Can you add a const to hold this value?